plugins/modules/rds_cluster_snapshot.py

```diff
@@ -0,0 +1,300 @@
+#!/usr/bin/python
+# Copyright (c) 2014 Ansible Project
+# Copyright (c) 2021 Alina Buzachis (@alinabuzachis)
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = r'''
+---
+module: rds_cluster_snapshot
+version_added: 3.2.0
+short_description: Manage Amazon RDS snapshots of DB clusters.
+description:
+     - Creates or deletes RDS snapshots of DB clusters.
+options:
+  state:
+    description:
+      - Specify the desired state of the snapshot.
+    default: present
+    choices: [ 'present', 'absent']
+    type: str
+  db_snapshot_identifier:
+    description:
+      - The identifier of the DB cluster snapshot.
+    required: true
+    aliases:
+      - snapshot_id
+      - id
```

Comment:
maybe we can also introduce `name` as an alias?

Reply:
,,,or snapshot_name? It feels more intuitive. But maybe I am alone here.

Reply:
internaly it uses `snapshot_name` as variable name :)

Reply:
I guess it makes sense to have a `snapshot_name`.